packages/e2e-test-utils/src/show-block-toolbar.js

```diff
@@ -3,7 +3,9 @@
  * Call this function to reveal it.
  */
 export async function showBlockToolbar() {
+	const content = await page.$( '.interface-interface-skeleton__content' );
+	const { x, y } = await content.boundingBox();
 	// Move the mouse to disable the isTyping mode
-	await page.mouse.move( 50, 50 );
-	await page.mouse.move( 100, 100 );
+	await page.mouse.move( x + 50, y + 50 );
+	await page.mouse.move( x + 100, y + 100 );
```

Comment:
Why this change was necessary?

Reply:
The rectangle of elements in the iframe will be relative to the iframe position. We need to offset that.

Reply:
But the idea here is just to move the mouse randomly to trigger the block toolbar, not sure we need precise position?

Reply:
With the iframe, the mouse needs to be over the canvas to trigger the block toolbar. Does that not make sense? I can change it so it works for the main window too.

Reply:
I restored the behaviour.